packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -251,7 +251,6 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
                         category,
                     },
                     error);
-                this.emit("summarizerWarning", error);
```

Comment:
If you're deleting all of the "summarizerWarning" it would also make sense to delete the consumption of the event in the container runtime.

`this.summaryManager.on("summarizerWarning", this.summarizerWarning);`